packages/medusa/src/api-v2/admin/products/route.ts

```diff
@@ -1,17 +1,17 @@
+import { createProductsWorkflow } from "@medusajs/core-flows"
+import { LinkModuleUtils } from "@medusajs/modules-sdk"
+import { CreateProductDTO } from "@medusajs/types"
+import { remoteQueryObjectFromString } from "@medusajs/utils"
 import {
   AuthenticatedMedusaRequest,
   MedusaResponse,
 } from "../../../types/routing"
 
-import { CreateProductDTO } from "@medusajs/types"
-import { createProductsWorkflow } from "@medusajs/core-flows"
-import { remoteQueryObjectFromString } from "@medusajs/utils"
-
 export const GET = async (
   req: AuthenticatedMedusaRequest,
   res: MedusaResponse
 ) => {
-  const remoteQuery = req.scope.resolve("remoteQuery")
+  const remoteQuery = req.scope.resolve(LinkModuleUtils.REMOTE_QUERY)
```

Comment:
question: Why don't we use `ContainerRegistrationKeys`, since when do we have that utils and since when the module SDK is aware of how the remote query is registered into an external container it does not know anything about? :) I believe this entire util can be removed entirely

Reply:
I would recommend to remove this enum everywhere it is used in favor of the container registration keys which was created for that purpose

Reply:
Ah, didn't know there was one already, will change em